app/assets/javascripts/discourse/app/components/sidebar.js

```diff
@@ -30,10 +31,43 @@ export default class Sidebar extends GlimmerComponent {
       this.args.toggleSidebar();
     }
   }
+  @bind
+  scrollToElement(destinationElement) {
+    const topPadding = 10;
```

Comment:
Is the value here dependent on somewhere else?

Reply:
no, this was my way of "no magic number" rule. So kind of config value at the top, but not used anywhere else, so don't need exported constant